zkwrapper/source/zkwrapper.cc

```diff
@@ -410,6 +342,29 @@ bool ZKWrapper::delete_node(const std::string &path, int &error_code) const {
 	return true;
 }
 
+bool ZKWrapper::get_info(const std::string &path,
+                         struct Stat &stat,
+                         int &error_code) const {
+
+    std::vector<std::uint8_t> data;
+    int len = MAX_PAYLOAD;
+    data.resize(len);
+
+    error_code = zoo_get(zh,
```

Comment:
We can't; ZKWrapper::get throws away the stat struct. Need to use zoo_get because get_info needs that stat struct.